app/js/lib/form_components.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+var objectAssign = require( 'object-assign' ),
+	actions = require( './actions' ),
+	createDefaultChangeHandler = function ( store, contentName ) {
+		return function ( evt ) {
+			store.dispatch( actions.newChangeContentAction( contentName, evt.target.value ) );
+		};
+	},
+	RadioComponent = {
+		element: null,
+		contentName: '',
+		onChange: null,
+		render: function ( formContent ) {
```

Comment:
Many JavaScript view libraries have a `render` method that signifies to the view/component that it should render itself. So the mismatch IMHO comes from the parameter name, not from the method name.